overlord/configstate/configcore/services.go

```diff
@@ -34,6 +34,7 @@ import (
 var services = []struct{ configName, systemdName string }{
 	{"ssh", "ssh.service"},
 	{"rsyslog", "rsyslog.service"},
+	{"backlight", "systemd-backlight@.service"},
```

Comment:
I know it's a service, but conceptually it's closer to system.power-key-action (see powerbtn.go), I would prefer if the option was something like "system.backlight-service" distinguished from these  other service options

Reply:
okay..I'll send a new PR to add the option.